tests/core/nlg/test_response.py

```diff
@@ -625,3 +625,29 @@ async def test_nlg_conditional_response_variations_condition_logging(
         "[condition 2] type: slot | name: test_B | value: B" in message
         for message in caplog.messages
     )
+
+
+async def test_nlg_response_with_jinja_template():
```

Comment:
let's also test 
* what happens if variables are referenced that do not exist
* if there is no `text` on the response
* that you can have a response using the default template mechanism and another template using jinja

Reply:
Thank you @tmbo, added the test scenarios 🙏